recipes/mastercard/create_mastercard.sql

```diff
@@ -0,0 +1,346 @@
+INSERT INTO mastercard.zip_to_borough values ('11416','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11373','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11371','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11361','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11414','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('10301','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10303','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10302','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10304','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10314','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10305','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10306','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10308','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10312','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10309','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10307','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10310','SI',5);
+END TRANSACTION;
+
+
+BEGIN TRANSACTION;
+CREATE TEMP TABLE tmp_csv (
+    yr numeric,
+    txn_date date,
+    industry text,
+    segment text,
+    geo_type text,
+    geo_name text,
+    nation_name text,
+    Zip_code text,
+    txn_amt double precision,
+    txn_cnt double precision,
+    acct_cnt double precision,
+    avg_ticket double precision,
+    avg_freq double precision,
+    avg_spend_amt double precision,
+    yoy_txn_amt text,
+    yoy_txn_cnt text
+);
+
+\COPY tmp_csv from PSTDIN DELIMITER '|' CSV HEADER;
+
+--process tmp_processed to remove nulls.
+/*
+DROP TABLE IF EXISTS tmp_processed
+CREATE TEMP TABLE tmp_processed (
+    yr numeric,
+    txn_date date,
+    industry text,
+    segment text,
+    geo_type text,
+    geo_name text,
+    nation_name text,
+    zip_code text,
+    txn_amt_index double precision,
+    txn_cnt_index double precision,
+    acct_cnt_index double precision,
+    avg_ticket_index double precision,
+    avg_freq_index double precision,
+    avg_spend_amt_index double precision,
+    yoy_txn_amt double precision,
+    yoy_txn_cnt double precision
+);
+*/
+
+
+CREATE SCHEMA IF NOT EXISTS :NAME;
+DROP TABLE IF EXISTS :NAME.:"VERSION" CASCADE;
+
+SELECT
+    tmp.yr as yr,
+    tmp.txn_date as txn_date,
+    tmp.industry as industry,
+    tmp.segment as segment,
+    tmp.geo_type as geo_type,
+    tmp.geo_name as geo_name,
+    tmp.nation_name as nation_name,
+    tmp.zip_code as zip_code,
+    tmp.txn_amt as txn_amt_index,
+    tmp.txn_cnt as txn_cnt_index,
+    tmp.acct_cnt as acct_cnt_index,
+    tmp.avg_ticket as avg_ticket_index,
+    tmp.avg_freq as avg_freq_index,
+    tmp.avg_spend_amt as avg_spend_amt_index,
+    (CASE 
+        WHEN tmp.yoy_txn_amt = 'None' THEN 0.0
+        ELSE CAST(tmp.yoy_txn_amt AS double precision)
+    END
+    ) as yoy_txn_amt,
+    (CASE
+        WHEN tmp.yoy_txn_cnt = 'None' THEN 0.0
```

Comment:
@SteveScott how many "None"s in raw data? We should check with mastercard what these mean.